src/datasources/db/database.options.ts

```diff
@@ -1,5 +1,6 @@
 import { Webhook } from '../../routes/webhook/entities/webhook.entity';
 import { DataSourceOptions } from 'typeorm';
+import * as fs from 'fs';
```

Comment:
You can import only the function you are interested in instead of the whole module by:
```javascript
import { readFileSync } from 'fs';
```